server/src/main/scala/com/metamx/tranquility/server/http/ServerMain.scala

```diff
@@ -52,18 +54,25 @@ import org.eclipse.jetty.util.thread.QueuedThreadPool
 import org.scala_tools.time.Imports._
 import org.skife.config.ConfigurationObjectFactory
 
-object Main
+object ServerMain extends App
 {
   private val ConfigResource = "tranquility-server.yaml"
 
-  def main(args: Array[String]): Unit = {
-    val configInputStream = if (args.isEmpty) {
-      getClass.getClassLoader.getResourceAsStream(ConfigResource) mapNull {
-        System.err.println(s"Expected resource $ConfigResource (or provide path on the command line)")
-        sys.exit(1)
-      }
-    } else {
-      new FileInputStream(args(0))
+  private val configFile: Flag[String] = flag(
+    "configFile",
+    s"Path to alternate config file, from working directory (default: $ConfigResource from classpath)"
+  )
+
+  def main(): Unit = {
+    val configInputStream = configFile.get match {
+      case Some(file) =>
+        new FileInputStream(file)
+
+      case None =>
+        getClass.getClassLoader.getResourceAsStream(ConfigResource) mapNull {
```

Comment:
Suggest logging information about the fact that you are pulling configs from the classpath resource.